File: schedule_management/lib/Models/Task.dart
import 'package:flutter/cupertino.dart';

class Task with ChangeNotifier {
  String id;
  bool done;
  String titleTask;
  String reminderTime;
  String lastEditedTime;

  Task empty() {
    return new Task('', false, '', '', '');
  }

  Task(this.id, this.done, this.titleTask, this.reminderTime,
      this.lastEditedTime);

  factory Task.fromJson(Map<String, dynamic> json) {
    print(json['done']);
    return Task(json['id'], json['done'], json['titleTask'],
        json['reminderTime'], json['lastEditedTime']);
  }
}

File: Cloud_API/models/Task.js
class Task {
    constructor(id, done, titleTask, reminderTime, lastEditedTime) {
        this.id = id;
        this.done = done;
        this.titleTask = titleTask;     this.reminderTime = reminderTime;
        this.lastEditedTime = lastEditedTime;
    }
}

module.exports = Task;